src/bonuses.cpp

```diff
@@ -58,6 +61,39 @@ affected_stat affected_stat_from_string( const std::string &s )
     return AFFECTED_NULL;
 }
 
+static const std::map<affected_stat, std::string> affected_stat_map_translation = {{
+        std::make_pair( AFFECTED_HIT, translate_marker( "Accuracy" ) ),
+        std::make_pair( AFFECTED_DODGE, translate_marker( "Dodge" ) ),
+        std::make_pair( AFFECTED_BLOCK, translate_marker( "Block" ) ),
+        std::make_pair( AFFECTED_SPEED, translate_marker( "Speed" ) ),
+        std::make_pair( AFFECTED_MOVE_COST, translate_marker( "Move cost" ) ),
+        std::make_pair( AFFECTED_DAMAGE, translate_marker( "damage" ) ),
+        std::make_pair( AFFECTED_ARMOR, translate_marker( "Armor" ) ),
+        std::make_pair( AFFECTED_ARMOR_PENETRATION, translate_marker( "Armor pen" ) ),
+        std::make_pair( AFFECTED_TARGET_ARMOR_MULTIPLIER, translate_marker( "Target armor multiplier" ) ),
+    }
+};
+
+std::string string_from_affected_stat( const affected_stat &s )
+{
+    const auto &iter = affected_stat_map_translation.find( s );
+    return iter != affected_stat_map_translation.end() ? iter->second : "";
```

Comment:
And now you need to actually translate the string (the marker above only register the string to be given to the translation team). Same in the function below.